libclamav_rust/build.rs

```diff
@@ -26,9 +26,25 @@ const LIB_LINK_WINDOWS: [&str; 4] = ["wsock32", "ws2_32", "Shell32", "User32"];
 // Windows library names that must have the leading `lib` trimmed (if encountered)
 const WINDOWS_TRIM_LOCAL_LIB: [&str; 2] = ["libclamav", "libclammspack"];
 
+const C_HEADER_OUTPUT: &str = "clamav_rust.h";
+
 fn main() {
+    detect_clamav_build();
+    execute_cbindgen();
+}
+
+fn execute_cbindgen() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+    cbindgen::generate(crate_dir)
+        .expect("Unable to generate bindings")
+        .write_to_file(C_HEADER_OUTPUT);
```

Comment:
There may be an environment variable we could use to set the path for where it writes the file.

Reply:
I tweaked `execute_cbindgen()` to check for `ARGS_WORKING_DIRECTORY`, but it still drops into the `libclamav_rust` directory.  Wherever it lands, it needs to be within the `-I` path `cc` is using.  Suggestions?

Reply:
Should be resolved now. Looks like the top of the build directory is already in the C include path.